<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAjoutTicket</class>
 <widget class="QWidget" name="PageAjoutTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Catégorie</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="categorie_box"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Système</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="systeme_box"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Logiciel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="logiciel_box"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Entrez votre message :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QTextEdit" name="message_text"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="creer_le_ticket_bouton">
     <property name="text">
      <string>Créer le ticket</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="bouton_retour">
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
